src/pages/iou/WaypointEditor.js

```diff
@@ -161,13 +190,9 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
 WaypointEditor.displayName = 'WaypointEditor';
 WaypointEditor.propTypes = propTypes;
 WaypointEditor.defaultProps = defaultProps;
-export default compose(
-    withLocalize,
-    withNetwork(),
-    withOnyx({
-        transaction: {
-            key: (props) => `${ONYXKEYS.COLLECTION.TRANSACTION}${props.transactionID}`,
-            selector: (transaction) => (transaction ? {transactionID: transaction.transactionID, comment: {waypoints: lodashGet(transaction, 'comment.waypoints')}} : null),
-        },
-    }),
-)(WaypointEditor);
+export default withOnyx({
+    transaction: {
+        key: (props) => `${ONYXKEYS.COLLECTION.TRANSACTION}${props.transactionID}`,
```

Comment:
Not a Blocker: you can de-structure here

```suggestion
        key: ({ transactionID }) => `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`,
```